src/phoenix/datasets/types.py

```diff
@@ -183,3 +208,196 @@ def __post_init__(self) -> None:
     Callable[[Example], TaskOutput],
     Callable[[Example], Awaitable[TaskOutput]],
 ]
+
+
+@dataclass(frozen=True)
+class ExperimentConfig:
+    n_examples: int
+    n_repetitions: int = 1
+
+    @property
+    def count(self) -> int:
+        return self.n_examples * self.n_repetitions
+
+
+@dataclass(frozen=True)
+class EvaluationConfig:
+    eval_names: FrozenSet[str]
+    exp_config: ExperimentConfig
+
+
+@dataclass(frozen=True)
+class _HasStats:
+    _title: str = field(repr=False, default="")
```

Comment:
title is the name of the experiment?

Reply:
it's the title for the summary printout

Reply:
i can add the experiment name to it

Reply:
actually don't have the experiment name in the payload. punt